products/memcache/api.yaml

```diff
@@ -0,0 +1,147 @@
+        'Official Documentation':
+          'https://cloud.google.com/memcache/docs/creating-instances'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        description: |
+          The name of the Memcache region of the instance.
+        required: true
+        input: true
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The resource name of the instance.
+        required: true
+        url_param_only: true
+        pattern: projects/{{project}}/locations/{{region}}/instances/{{name}}
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        description: |
+          A user-visible name for the instance.
+      - !ruby/object:Api::Type::String
+        name: 'state'
+        description: |
+          The instance state - short description.
+        output: true
+        exclude: true
+      - !ruby/object:Api::Type::Array
+        name: 'instanceMessages'
+        description: |
+          Additional information about the instance state, if available.
+        output: true
+        exclude: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'code'
+              description: An error code.
+            - !ruby/object:Api::Type::String
+              name: 'message'
+              description: The message to be displayed to a user.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: Creation timestamp in RFC3339 text format.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          Resource labels to represent user-provided metadata.
+      - !ruby/object:Api::Type::Array
+        name: 'zones'
+        description: |
+                Zones where memcache nodes should be provisioned.  If not
+                provided, all zones will be used.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::String
+        name: 'authorizedNetwork'
+        description: |
+          The full name of the GCE network to connect the instance to.  If not provided,
+          'default' will be used.
+      - !ruby/object:Api::Type::Integer
+        name: nodeCount
+        description: |
+          Number of nodes in the memcache instance.
+        required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: nodeConfig
+        description: |
+          Configuration for memcache nodes.
+        required: true
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: cpuCount
+            description: |
+              Number of CPUs per node.
+            required: true
+          - !ruby/object:Api::Type::Integer
+            name: memorySizeMb
+            description: |
+              Memory size in Mibibytes for each memcache node.
```

Comment:
"Mibibytes"?

Reply:
Yep - MiB.

https://en.wikipedia.org/wiki/Mebibyte

Reply:
Oh I guess it's "mebibytes" - typo'd it.